inference-engine/src/gna_plugin/gna_graph_compiler.cpp

```diff
@@ -325,6 +325,10 @@ void GNAGraphCompiler::ConvolutionPrimitive(InferenceEngine::CNNLayerPtr layer)
     uint32_t num_columns_out = (((num_inputs + num_input_padding - num_filter_coefficients) / num_feature_map_columns) + 1) * convolution._out_depth;
     uint32_t num_columns_out_unpadded = (((num_inputs - single_conv_kernel_size) / num_feature_map_columns) + 1) * convolution._out_depth;
 
+    if(num_columns_out != num_columns_out_unpadded) {
```

Comment:
i think depends on implementation we cannot extend num_columns_out in arbitrary way - please add more constraint.

Reply:
removed this statement and incorporated it to while below